docs/DeveloperGuide.adoc

```diff
@@ -320,8 +320,8 @@ _{More to be added}_
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
+1.  User requests to list people
```

Comment:
It seems that the developer guide is about the Address Book, not our software. Could you update the contents of the developer's guide to be specific to our software?